lib/vsc/mympirun/option.py

```diff
@@ -123,8 +123,7 @@ def make_init(self):
 
             "showsched": ("Print the known Sched classes and exit", None, "store_true", False, 's'),
 
-
-            "socket": ("Force socket device", None, "store_true", None),
+            "sockets-per-node": ("Number of sockets per cpu (default/0: get from /proc/cpuinfo)", "int", "store", 0),
```

Comment:
`s/per cpu/per node/g` (a CPU is in a socket ;))

also, clarify w.r.t. default: `(default: 0, i.e. try to detect # sockets via /proc/cpuinfo)`